core/src/block.rs

```diff
@@ -913,15 +907,11 @@ pub mod stream {
         }
     }
 
-    /// Message sent by the stream consumer
+    /// Message sent by the stream consumer.
+    /// Request sent to subscribe to blocks stream starting from the given height.
     #[version_with_scale(n = 1, versioned = "VersionedBlockSubscriberMessage")]
-    #[derive(Debug, Clone, Copy, Decode, Encode, FromVariant, IntoSchema)]
-    pub enum BlockSubscriberMessage {
-        /// Request sent to subscribe to blocks stream starting from the given height.
-        SubscriptionRequest(u64),
-        /// Acknowledgment of receiving block sent from the peer.
-        BlockReceived,
-    }
+    #[derive(Debug, Clone, Copy, Decode, Encode, IntoSchema)]
+    pub struct BlockSubscriberMessage(pub u64);
```

Comment:
consider renaming to `SubscriptionRequest`

Reply:
I will use `BlockSubscriptionRequest`